### ISSUE TITLE
Bugnsag deploy notification failed, #<NoMethodError: undefined method `configuration' for Bugsnag:Module>

### ISSUE THREAD

**Issue by antoinelyset** [bugsnag/bugsnag-ruby#274]
Hi Bugsnag,

Thanks for your work, we are trying to integrate the Capistrano hook with : 

```ruby
require 'bugsnag/capistrano'
set :bugsnag_api_key, 'API-KEY'
```
It doesn't seem to work, we got the error : 

```
Bugnsag deploy notification failed, #<NoMethodError: undefined method `configuration' for Bugsnag:Module>
```

We use capistrano (2.15.5) and bugsnag (3.0.0).

**Comment by kattrali** [comment]
Fixed by #275!

**Comment by antoinelyset** [comment]
Thanks !
